lib/ardb/adapter/base.rb

```diff
@@ -7,8 +7,8 @@ class Base
     attr_reader :schema_format, :ruby_schema_path, :sql_schema_path
 
     def initialize
-      @config_settings = Ardb.config.db_settings
-      @database = Ardb.config.db.database
+      @config_settings = Ardb.config.activerecord_connect_hash
```

Comment:
@kellyredding - Yeah something like that. I chose not to change it here and leave it for when I modify the adapters more but I can change it here. It probably makes more sense to do it here.